packages/api/tests/custom_animal_type.test.js

```diff
@@ -48,17 +48,14 @@ describe('Custom Animal Type Tests', () => {
     done();
   });
 
-  function getRequest({ user_id, farm_id }, callback) {
-    chai
+  async function getRequest({ user_id, farm_id }, query = '') {
+    return await chai
       .request(server)
-      .get('/custom_animal_types')
+      .get(`/custom_animal_types${query}`)
       .set('user_id', user_id)
-      .set('farm_id', farm_id)
-      .end(callback);
+      .set('farm_id', farm_id);
   }
 
-  const getRequestAsPromise = util.promisify(getRequest);
```

Comment:
Thank you for looking into this! I had always copied it as boilerplate without questioning whether it was necessary.

Reply:
I had errors in my tests and blamed this, but they were just due to my mistakes...
However, ChatGPT says this is not used how it is supposed to be 😂 I think we can stop copying this in the future!